### ISSUE TITLE
[MU4 Issue] Musescore 4 crashes after selecting a specific bar

### ISSUE THREAD

**Issue by Xnthr1** [musescore/MuseScore#15619]
**Describe the bug**
When I select specific bar(s) or note(s) in score, it crashes MuseScore
I am using Linux Mint 20.3 Una

**To Reproduce**
Not sure how to reproduce on different devices.
When running through the terminal and selecting that bar it shows
`[51882:51882:20221227,121628.069986:ERROR file_io_posix.cc:144] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq: No such file or directory (2)
[51882:51882:20221227,121628.070012:ERROR file_io_posix.cc:144] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq: No such file or directory (2)
12:16:28.076 | INFO  | main_thread | ActionsDispatcher | dispatch: try call action: note-a
/tmp/.mount_MuseScNoCR46/AppRun: line 26: 51874 Aborted                 (core dumped) "${APPDIR}/bin/mscore4portable" "$@"
`

**Expected behavior**
It shouldn't have crashed

**Screenshots**

**Platform information**
 - OS: Linux Mint 20.3 Una

**Additional context**
This musescore file, when I used it a few months in 3.6.2 would come up with "This file is corrupted" errors and these specific bars would the ones that were corrupted. I fixed it by deleting the bar and recreating it.

**Comment by DmitryArefiev** [comment]
@Xnthr1 Is it occur with any score or specific?  Does playback work?

**Comment by Xnthr1** [comment]
Its just this one and playback works but there there are some notes around the area that doesn't play the full duration.

**Comment by DmitryArefiev** [comment]
> Its just this one and playback works but there there are some notes around the area that doesn't play the full duration.
> 
> Edit, I tried this again this morning and it completely crashed. [https://youtu.be/ZQKD6KVyDlk](url)

Oh.. can you attach the score please? (Or if you want can send it directly to me)

**Comment by Xnthr1** [comment]
How do I upload the Musescore file here or send it to you? It doesn't support Musescore files

**Comment by cbjeukendrup** [comment]
You can zip the `.mscz` file, or just rename it to replace `.mscz` with `.zip`, since a `.mscz` file is under the hood a kind of zip file.

**Comment by Xnthr1** [comment]
[Third_Date_(HTTYD3_String_Orchestra_Cover) (copy).zip](https://github.com/musescore/MuseScore/files/10315173/Third_Date_.HTTYD3_String_Orchestra_Cover.copy.zip)
Here you go

**Comment by J12345Z-H** [comment]
No, sir. I'm selecting things and it's not crashing on my Chrome OS.

**Comment by J12345Z-H** [comment]
For what it's worth, nice score too.

**Comment by Xnthr1** [comment]
> 

This might be because it's a system issue, just not sure what to fix.

**Comment by DmitryArefiev** [comment]
> [Third_Date_(HTTYD3_String_Orchestra_Cover) (copy).zip](https://github.com/musescore/MuseScore/files/10315173/Third_Date_.HTTYD3_String_Orchestra_Cover.copy.zip) Here you go

@Xnthr1 Now, I **can reproduce** it on my **Ubuntu** with enabled MuseSounds when **click F note in Violins 2 in measure 89**. 

Thanks for the score! I will add steps to the description
